release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -51,6 +51,7 @@ def set_cluster_env(self, cluster_env: Dict[str, Any]):
         self.cluster_env["env_vars"]["RAY_bootstrap_with_gcs"] = "1"
         self.cluster_env["env_vars"]["RAY_USAGE_STATS_ENABLED"] = "1"
         self.cluster_env["env_vars"]["RAY_USAGE_STATS_SOURCE"] = "nightly-tests"
+        self.cluster_env["env_vars"]["RAY_task_oom_retries"] = "15"
```

Comment:
Can we avoid this? It's ideal we are using the default configuration if possible (I prefer not to have retries 15 only for nightly tests)